benchmarks/base64/benchmark_base64.cpp

```diff
@@ -110,10 +112,13 @@ std::vector<char> read_file(const char *filename,
 void show_help() {
   printf("Usage: benchmark_base64 [options] file1 [file2 ...]\n");
   printf("Options:\n");
-  printf("  -h, --help     Show this help message and exit\n");
-  printf("  -d, --decode   Decode the input file\n");
-  printf("  -e, --encode   Encode the input file\n");
+  printf("  -h, --help        Show this help message and exit\n");
+  printf("  -d, --decode      Decode the input file\n");
+  printf("  -e, --encode      Encode the input file\n");
   printf("  -r, --roundtrip   Roundtrip the input file\n");
+  printf("  --roundtripurl    Roundtrip the input file (URL)\n");
```

Comment:
Small suggestion that needs more than the suggested code change:

```suggestion
  printf("  --roundtrip-url    Roundtrip the input file (URL)\n");
```